mailpoet/mailpoet
# Measure unsubscribed by type [MAILPOET-4733]

## Description

This PR adds different methods to unsubscribe records based on method and exposes these stats to the reporter.

## Code review notes

_N/A_

## QA notes

Verify the following are exposed:
```
'Unsubscribe > Total in last 6 months'
'Unsubscribe > Count in last 6 months with method: Link'
'Unsubscribe > Count in last 6 months with method: 1 Click'
```

## Linked tickets

[MAILPOET-4733]


[MAILPOET-4733]: https://mailpoet.atlassian.net/browse/MAILPOET-4733?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
